packages/frontend/src/hooks/echarts/useEcharts.ts

```diff
@@ -633,12 +650,21 @@ const getSimpleSeries = ({
             ...series.label,
             ...(formats &&
                 formats[yFieldHash] && {
-                    formatter: (value: any) =>
-                        formatValue(value?.value?.[yFieldHash], {
-                            format: formats[yFieldHash].format,
-                            round: formats[yFieldHash].round,
-                            compact: formats[yFieldHash].compact,
-                        }),
+                    formatter: (value: any) => {
+                        const field = formats[yFieldHash];
+                        if ('round' in field) {
```

Comment:
`round` is an optional property in `Field` type. This conditional doesn't feel safe has might have a lot of fields ending up in the `else` block. What do you think ?